.gitignore

```diff
@@ -9,5 +9,4 @@ cache
 artifacts
 
 oracles/node_modules
-package-lock.json
```

Comment:
Left a note in the PR description about this:

FYI I noticed we did not check in the `package-lock.json` files. These should be checked into the repo, so that we are working with the same dependency trees. I removed this entry from `.gitignore`, so that they are included. I would recommend running `npm ci` in `client` and `protocol` once this PR is merged.